### PR TITLE
Expand and revise Submariner doc in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26051

I ended up disagreeing with changing the command to a manifest example as there are plenty of commands throughout the doc and there is no reason to use a longer manifest example when a shorter command does the same thing. We also don't do central manifests as far as I know. I did implement the other suggestions. Everybody please help review, thanks!

### CODE REVIEW

**Review by cadawson — COMMENTED** [review]
This looks good, Oliver. Let's see what Daniel thinks about merging the Cable driver options in one example. Either way works. If you think it is a case that is different enough to have a separate example, I am good with that.

File: add-ons/submariner/submariner_customizations.adoc [hunk 1]
```diff
@@ -135,3 +135,22 @@ spec:
 
 *Note:* The name of the `SubmarinerConfig` must be `submariner`, as shown in the example.
 
+[#cable-driver]
+== Cable driver
+
+The Submariner gateway engine component creates secure tunnels to other clusters. You can use the Libreswan or VXLAN implementations for the cable engine component for your `cableDriver` configuration. See the following example:
+
+[source,yaml]
+----
```
Comment (MaxBab) [inline add-ons/submariner/submariner_customizations.adoc:144]: It looks like the example of yaml file missing indentation.

File: add-ons/submariner/submariner_uninstall.adoc [hunk 2]
```diff
@@ -29,23 +31,23 @@ If the version of Submariner that you are removing is earlier than version 0.12,
 
 To uninstall Submariner by using the command line, complete the following steps:
 
-. Locate the clusters that contain the Submariner add-on by entering the following command:
+. Remove the Submariner deployment for the cluster by running the following command:
 +
 ----
-oc get resource submariner-addon -n open-cluster-management
+oc -n get managedclusteraddon submariner
```
Comment (MaxBab) [inline add-ons/submariner/submariner_uninstall.adoc]: - The `managedclusteraddon` resource should be deleted, not just fetched.
- The `-n` argument should define the `<managed-cluster-namespace>`.

`oc -n <managed-cluster-namespace> delete managedclusteraddon submariner`

File: add-ons/submariner/submariner_uninstall.adoc [hunk 3]
```diff
@@ -29,23 +31,23 @@ If the version of Submariner that you are removing is earlier than version 0.12,
...
 ----
 
-. Run a command similar to the following example to uninstall Submariner from the cluster:
+. Remove the cloud resources of the cluster by running the following command:
 +
 ----
-oc delete resource submariner-addon -n <CLUSTER_NAME>
+oc -n delete submarinerconfig submariner
```
Comment (MaxBab) [inline add-ons/submariner/submariner_uninstall.adoc]: The `-n` argument should define the `<managed-cluster-namespace>`

File: add-ons/submariner/submariner_uninstall.adoc [hunk 4]
```diff
@@ -29,23 +31,23 @@ If the version of Submariner that you are removing is earlier than version 0.12,
...
-+
-Replace `CLUSTER_NAME` with the name of the cluster.
-
-. Confirm that you want to remove all of the Submariner components from the cluster. 
 
-. Repeat the steps for each cluster to remove Submariner.
+. Delete the cluster set to remove the broker details by running the following command:
++
+----
+oc delete managedclusterset
```
Comment (MaxBab) [inline add-ons/submariner/submariner_uninstall.adoc]: The command should define the name of the clusterset that should be deleted.

File: add-ons/submariner/submariner_customizations.adoc (hunk 1 above)
Comment (oafischer) [reply add-ons/submariner/submariner_customizations.adoc:144]: Fixed

File: add-ons/submariner/submariner_uninstall.adoc (hunk 2 above)
Comment (oafischer) [reply add-ons/submariner/submariner_uninstall.adoc]: Fixed

File: add-ons/submariner/submariner_uninstall.adoc (hunk 3 above)
Comment (oafischer) [reply add-ons/submariner/submariner_uninstall.adoc]: Fixed

File: add-ons/submariner/submariner_uninstall.adoc (hunk 4 above)
Comment (oafischer) [reply add-ons/submariner/submariner_uninstall.adoc]: Fixed

**Comment by oafischer** [comment]
@dfarrell07 I've made some changes based on @MaxBab's comments, can you take one last look? Thanks!

File: add-ons/submariner/submariner_uninstall.adoc [hunk 5]
```diff
@@ -29,23 +31,29 @@ If the version of Submariner that you are removing is earlier than version 0.12,
...
-Replace `CLUSTER_NAME` with the name of the cluster.
+Replace `managed-cluster-namespace` with the namespace of your managed cluster.
 
-. Confirm that you want to remove all of the Submariner components from the cluster. 
-
-. Repeat the steps for each cluster to remove Submariner.
+. Delete the cluster set to remove the broker details by running the following command:
++
+----
+oc delete <managedclusterset>
```
Comment (MaxBab) [inline add-ons/submariner/submariner_uninstall.adoc]: Small mistake in the cluster set deletion command.

Should be:
`oc delete managedclusterset <managed-clusterset-name>`

Comment (oafischer) [reply add-ons/submariner/submariner_uninstall.adoc]: Fixed! Thanks for catching that

**Review by MaxBab — APPROVED** [review]
Looks good.

**Comment by oafischer** [comment]
This needs to merge for a refresh so I'm going ahead with one dev approval and writing approval.

**Pull request merged**